ClassevivaPCTO/Controls/NoticesListView.xaml

```diff
@@ -56,18 +56,10 @@
                                Text="{x:Bind Category}"
                                TextWrapping="NoWrap" />
 
-                    <TextBlock x:Name="dataPubblicazione" Margin="0,0,0,0"
```

Comment:
perchè togliere la data di pubblicazione?

Reply:
lo includiamo sotto, così diventa

_data • valida fino a data2_

e risparmiamo spazio

Reply:
in molti casi nella mia scuola la data di pubblicazione era diversa dalla data di inizio validità, togliendola rimuoviamo una informazione che può essere importante

Reply:
allora la mettiamo sotto e la separiamo dalla validità col pallino